### DISCUSSION TITLE
支持泛解析吗？

### DISCUSSION THREAD

**Discussion by yigehaozi** [jeessy2/ddns-go#1167] [General]
支持泛解析吗？

**Comment by jeessy2** [comment]
可以

**Comment by yigehaozi** [comment]
要怎么操作
---- 回复的原邮件 ----
***@***.***>发送日期2024年06月23日 10:04 ***@***.***> ***@***.***>,
***@***.***>主题Re: [jeessy2/ddns-go] 支持泛解析吗？ (Discussion #1167)
可以

**Reply by jeessy2** [reply]
前面加*.
*.example.com

**Reply by yigehaozi** [reply]
ok，谢谢我试试
